app/components/dj-donate-modal.hbs

```diff
@@ -0,0 +1,25 @@
+<a {{action "toggleDjDonateModal" }} ...attributes class="donate-button">
```

Comment:
Not a big deal but this is the new action syntax: :hammer_and_wrench: 
```suggestion
<a {{on "click" this.toggleDjDonateModal}} ...attributes class="donate-button">
```

Reply:
Noted and fixed, thanks!